packages/compiler/src/init/init-template.ts

```diff
@@ -31,7 +31,7 @@ export interface InitTemplate {
   /**
    * List of libraries to include
    */
-  libraries: string[];
+  libraries: InitTemplateLibrary[];
 
```

Comment:
I think we need to keep string as an option too so we don’t break existing templates

Reply:
Nvm see it’s the union below